### ISSUE TITLE
Problem with database download

### ISSUE THREAD

**Issue by lokyimandy** [jtamames/SqueezeMeta#610]
Hello, I tried download_databases.pl and test_install.pl returned an error "The LCA_tax/taxid.db database is not present in /download/path/db, it is malformed, or there is other problem with your SQLite configuration". I have checked LCA_tax directory and it shows :
0 Jan 18 22:30 parents.db
324M Jul 23 05:11 parents.txt
29G Jul 24 03:10 taxid.db
0 Jan 18 22:30 taxid.md5

Also, when I run the test samples with "SqueezeMeta.pl -m coassembly -p test -s test.samples -f raw --nopfam -miniden 50", the program died at step 4 with the error shown below:
[10 seconds]: STEP4 -> HOMOLOGY SEARCHES: 04.rundiamond.pl
perl: warning: Setting locale failed.
perl: warning: Please check that your locale settings:
	LANGUAGE = (unset),
	LC_ALL = (unset),
	LC_CTYPE = "UTF-8",
	LANG = "en_US.UTF-8"
    are supported and installed on your system.
perl: warning: Falling back to a fallback locale ("en_US.UTF-8").
  Setting block size for Diamond
  AVAILABLE (free) RAM memory: 183.28 Gb
  We will set Diamond block size to 16 (Gb RAM/8, Max 16).
  You can override this setting using the -b option when starting the project, or changing
  the $blocksize variable in SqueezeMeta_conf.pl
 taxaError: Unexpected end of input.
Error running command:    /path/miniconda3/envs/env_mamba/envs/SqueezeMeta/SqueezeMeta/bin/diamond blastp
 -q /path/SqueezeMeta1.6/metag_stw/results/03.metag_stw.faa -p 40 -d /path/SqueezeMeta1.6/db/nr.dmnd -e 0.001 --id 40 -f tab -b 16 --quiet -o /path/SqueezeMeta1.6/metag_stw/intermediate/04.metag_stw.nr.diamond at /path/miniconda3/envs/env_mamba/envs/SqueezeMeta/SqueezeMeta/scripts/04.rundiamond.pl line 72.
Died at /path/miniconda3/envs/env_mamba/envs/SqueezeMeta/bin/SqueezeMeta.pl line 921.

Since I am not the root user of the cluster, I created an environment with mamba installed(env_mamba), then use mamba create the environment of SqueezeMeta.

Would you please help with the issues? Thank you very much.

**Comment by lokyimandy** [comment]
I tried make_databases.pl as well, unfortunately the result seems to be worse, LCA_tax contains null files and even nr.db is corrupted.

**Comment by fpusan** [comment]
Hi, it seems that your download got interrupted at some point, or you ran out of space when running `download_databases.pl`.
Check that you have enough space and try running `download_databases.pl` again. Monitor the process to see if it reports errors at some point.
In version 1.6.1 (which we hope to release before the end of January) we include extra checks to make sure that database generation and download is progressing correctly.

**Comment by vmevada102** [comment]
Can you provide direct link to download all databases?

The download is very slow with the command to download the prebuilt database.

**Comment by fpusan** [comment]
Yes, see #484

**Comment by AlexandreThibodeauUdM** [comment]
Advice: have 600 Go of free space to install SqueezeMeta and the databases. I had problems with 500 Go on my virtual machine and no problems at all using 600 Go. I run my virtual machine with 26 Go of ram.
Cheers!

**Comment by lokyimandy** [comment]
Thank you all for your help. Yes, indeed it seems to be the free space problem. I installed the database with around 500 Gb of free space at the first time, it was not successful although there were still around 200 Gb left after the download. Then I tried to download it with around 800 Gb, I passed the install check and diamond step. Everything seems alright till now, but the LCA step take quite long (around 30 hr now still ongoing), hope I can finish it without any problems!
